core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -245,8 +245,10 @@ object ConsumerGroupCommand extends Logging {
     private def printState(group: String, state: Option[GroupState]): Unit = {
       // this method is reachable only for the new consumer option (where the given state is always defined)
       if (shouldPrintMemberState(group, Some(state.get.state), Some(1))) {
-        print("\n%-20s %-25s %-20s %s".format("COORDINATOR", "ASSIGNMENT-STRATEGY", "STATE", "#MEMBERS"))
-        print("\n%-20s %-25s %-20s %s".format(state.get.coordinator.id, state.get.assignmentStrategy, state.get.state, state.get.numMembers))
+        val coordinator = state.get.coordinator.host + ":" + state.get.coordinator.port + " (" + state.get.coordinator.idString + ")"
```

Comment:
nit: since we're in scala, maybe we can use string interpolation. Also, doesn't the `state.get` seem unsafe? Are we missing a `match`?

Reply:
Unlike `collectGroupOffsets` and `collectGroupMembers`, `collectGroupState` always returns content. I guess it makes more sense to change the output from `Option[GroupState]` to `GroupState`. What do you think?

Reply:
Yeah, if the `None` case is not actually possible, then we should not use `Option`.

Reply:
The new commit has the change I mentioned above.